bindings/python/test/test_arrow.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# from datetime import datetime, timedelta
+import unittest
+import unittest.mock as mock
+
+from pyarrow import int32, int64, schema as ArrowSchema, Table
+from pymongo import WriteConcern, DESCENDING
+import pymongo
+from pymongoarrow import aggregate_arrow_all, find_arrow_all, Schema
+
+from test import client_context
+from test.utils import WhiteListEventListener
```

Comment:
WhiteListEventListener -> AllowListEventListener

Reply:
Oof! Thanks for catching that.